Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroLayout/Samples/Ili9488Charts_Sample/Ili9488Charts_Sample.csproj

```diff
@@ -15,10 +15,4 @@
     <ProjectReference Include="..\..\Driver\Graphics.MicroLayout.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
And here - intentional delete?

Reply:
Yes.  libmpsse.dll is no longer required for FT232, so it's just removing library cruft